src/main/java/com/coinpaprika/apiclient/repository/coin/CoinApi.kt

```diff
@@ -12,12 +12,12 @@ class CoinApi constructor(
         .create(CoinApiContract::class.java)
 ) : CoinApiContract {
 
-    override fun getCoin(id: String): Observable<Response<CoinDetailsEntity>> {
-        return safeApiCallRaw { retrofit.getCoin(id) }
+    override suspend fun getCoin(id: String): CoinDetailsEntity {
```

Comment:
tutaj i niżej zniknęły Response, nie potrzebujemy tego?

Reply:
Wyglada na to, ze nie.  Zmiany ktore tu sa, sa wymuszone tym jak ten kod jest uzywany w innych modulach (`app` glownie) i `Response` nie jest tam wymagany.